pallets/common/src/traits/nft.rs

```diff
@@ -38,6 +38,9 @@ decl_event!(
         IssuedNFT(IdentityId, NFTCollectionId, NFTId),
         /// Emitted when an NFT is redeemed.
         RedeemedNFT(IdentityId, Ticker, NFTId),
+        /// NFTs successfully transferred.
+        /// Contains the sender's DID, the ticker for the collection, the id for each transferred NFT, the sender and receiver portfolios and the InstructionId.
+        NFTsTransferred(IdentityId, NFTs, PortfolioId, PortfolioId, u64),
```

Comment:
We should consider moving the types defined in settlement pallet to primitives.

Reply:
I just noticed that we don't include the `PortfolioId` in the `IssuedNFT` and `RedeemedNFT` events.  Since  those can be done with none default portfolios, they can't be used to track NFTs and portfolios.

Reply:
I created a ticket for the Issue and Redeem events: `MESH-2037`.